src/server_manager/web_app/ui_components/outline-manual-server-entry.js

```diff
@@ -188,6 +202,10 @@ Polymer({
               <iron-icon id="gcpCreateServerDropDownIcon" icon="arrow-drop-down"></iron-icon>
             </div>
           </div>
+          <div id="gcp-new-flow-promo" class="section-content-instructions" on-tap="gcpNewFlowTapped">
```

Comment:
Please move this to above the first card. It shouldn't look as part of the manual flow, but a escape hatch.

Reply:
Done